app/views/govuk_publishing_components/components/_select.html.erb

```diff
@@ -2,16 +2,16 @@
   options ||= []
   id ||= false
   label ||= false
+
+  select_helper = GovukPublishingComponents::Presenters::SelectHelper.new(local_assigns)
```

Comment:
Are we able to just pass options? I don't think anything else (id and label) is needed.